### PR TITLE
Feature/gui comm mgr

### PR DESCRIPTION
**Pull request by murgui**
Ladies and gentlemen, GUI staright from the oven.

### CODE REVIEW

**Review by NicolaiFransen — APPROVED** [review]
Looks really nice, ship it¿?

File: Software/Development/SemesterProject/System/Include/scheduler.h [hunk 1]
```diff
@@ -35,8 +36,12 @@ typedef struct TCBstruct
 void taskListInitialization(void);
 void scheduleTasks(void);
 void runTask(void (*functionPTR)());
+
 void task50us(void);
 void task20ms(void);
+void task200ms(void);
```
Comment (AitorTeran) [inline Software/Development/SemesterProject/System/Include/scheduler.h:45]: Is this the minimum freq at which the GUI can get  updated?

**Review by AitorTeran — APPROVED** [review]
Ship it!

File: Software/Development/SemesterProject/System/Include/scheduler.h (hunk 1 above)
Comment (murgui) [reply Software/Development/SemesterProject/System/Include/scheduler.h:45]: yes-ish. Maybe further research shows that it can be faster.

**Pull request merged**